force-app/main/shared/classes/EventHandlerTest.cls

```diff
@@ -0,0 +1,99 @@
+@isTest
+private without sharing class EventHandlerTest {
+
+	@testSetup
+	private static void setup() {
+        TAG_TestDataFactory.assignPermissionSet( 'accountTypeHandler@nav.test', 'ManageAccounts' );
+        }
+    
+    @isTest
+	private static void testOnAfterInsert() {
+
+        List<Account> acc = TAG_TestDataFactory.createAccounts( 1 );
+        Contact c = new Contact( TAG_Informed__c = true, LastName = 'test' );
+        
+        Test.StartTest();
+        Event e = new Event( WhatId = acc[0].Id, WhoId = c.Id, TAG_NoPersonInformation__c = true, Subject = 'testSubject', TAG_ActivityType__c = 'Rekruttere og inkludere', StartDateTime = System.Now(), EndDateTime = System.Now().addHours(1) );
+        insert e;
+        Test.StopTest();
+
+        List<EventShadow__c> es = [SELECT Id, SH_EventId__c FROM EventShadow__c];
+        Integer EventShadowCount = es.size() ;
+
+		System.assertEquals( 1, EventShadowCount );
+		System.assertEquals( e.Id, es[0].SH_EventId__c );
+
+    }
+    
+    @isTest
+	private static void testOnAfterUpdate() {
+
+        List<Account> acc = TAG_TestDataFactory.createAccounts( 1 );
+        Contact c = new Contact( TAG_Informed__c = true, LastName = 'test' );
+        Event e = new Event( WhatId = acc[0].Id, WhoId = c.Id, TAG_NoPersonInformation__c = true, Subject = 'testSubject', TAG_ActivityType__c = 'Rekruttere og inkludere', StartDateTime = System.Now(), EndDateTime = System.Now().addHours(1) );
+        insert e;
+        
+        Test.StartTest();
+        e.Subject = 'Subject Updated';
+        update e;
+        Test.StopTest();
+
+        List<EventShadow__c> es = [SELECT Id, SH_EventId__c, SH_Subject__c FROM EventShadow__c];
+        Integer EventShadowCount = es.size() ;
+
+		System.assertEquals( 1, EventShadowCount );
+        System.assertEquals( e.Id, es[0].SH_EventId__c );
+        System.assertEquals( 'Subject Updated', es[0].SH_Subject__c );
+
+    }
+    
+    @isTest
+	private static void testOnAfterDelete() {
+
+        List<Account> acc = TAG_TestDataFactory.createAccounts( 1 );
+        Contact c = new Contact( TAG_Informed__c = true, LastName = 'test' );
+        Event e = new Event( WhatId = acc[0].Id, WhoId = c.Id, TAG_NoPersonInformation__c = true, Subject = 'testSubject', TAG_ActivityType__c = 'Rekruttere og inkludere', StartDateTime = System.Now(), EndDateTime = System.Now().addHours(1) );
+        insert e;
+        List<EventShadow__c> es = [SELECT Id FROM EventShadow__c];
+        Integer EventShadowCount = es.size() ;
+
```

Comment:
Kjører en system.assert her FØR testen, for å verifisere at det finnes én record FØR og så én record ETTERPÅ